### PR TITLE
[CDS-101265] Updated Gitops Flux application

### PR DESCRIPTION
**Pull request by vishal-av**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-delivery/gitops/connect-and-manage/use-flux.md [hunk 1]
```diff
@@ -13,12 +13,169 @@ When you create a Flux application, to make the application available in the Git
 
 Before you can manage Flux applications from the Harness GitOps user interface, complete the following tasks:
```
Comment (mteodor) [inline docs/continuous-delivery/gitops/connect-and-manage/use-flux.md:14]: I would first list Prerequisites
and then explain steps that are required to satisfy prerequiestes

also this section doesnt seem quite correct
```When you create a Flux application, to make the application available in the GitOps user interface, you only need to specify that the application is of type *Flux* and provide the existing Flux manifests. You also need to select Flux as the sync policy so that Flux becomes the GitOps reconciler for that application. For more information about how FSA works, go to [Flux Subsystem for Argo](https://flux-subsystem-argo.github.io/website/).```

File: docs/continuous-delivery/gitops/connect-and-manage/use-flux.md [hunk 2]
```diff
@@ -13,12 +13,169 @@ When you create a Flux application, to make the application available in the Git
...
+
+Replace `FLUX_APP_NS` with the namespace where you want to deploy your application.
+
+Now you can see the flux application is created and is in sync.
 
-To enable Harness to manage a Flux application, do the following:
-* When creating a GitOps agent, select Flux as the GitOps operator. You configure all other configuration settings in the same way as you configure a Harness GitOps agent. For more information, go to [Install a Harness GitOps Agent](/docs/continuous-delivery/gitops/connect-and-manage/install-a-harness-git-ops-agent).
-* When creating an application, select Flux as the GitOps operator. You configure all other configuration settings in the same way as you configure any GitOps application. For more information, go to [Harness CD GitOps tutorial](/docs/continuous-delivery/gitops/get-started/harness-cd-git-ops-quickstart).
\ No newline at end of file
+![](./static/gitops-flux-6.png)
```
Comment (mteodor) [inline docs/continuous-delivery/gitops/connect-and-manage/use-flux.md:181]: having red exclamation sign od doc is not good

File: docs/continuous-delivery/gitops/connect-and-manage/use-flux.md [hunk 3]
```diff
@@ -13,12 +13,169 @@ When you create a Flux application, to make the application available in the Git
...
+
+![](./static/gitops-flux-3.png)
+
+In the Source tab, configure your repository with your flux application.
+
+![](./static/gitops-flux-4.png)
+
+In the destinations tab, select the Gitpos agent that you earlier created.
+
+![](./static/gitops-flux-5.png)
```
Comment (mteodor) [inline docs/continuous-delivery/gitops/connect-and-manage/use-flux.md:163]: it says `agrocd` I guess it was suppose to be `argocd`

File: docs/continuous-delivery/gitops/connect-and-manage/use-flux.md [hunk 4]
```diff
@@ -13,12 +13,169 @@ When you create a Flux application, to make the application available in the Git
...
+
+To generate a Flamingo application to visualize the Podinfo objects, run:
+
+```bash
+flamingo generate-app \
+  --app-name=podinfo-ks \
+  -n podinfo-kustomize ks/podinfo
+```
+
+Now, navigate to Harness GitOps Applications tab, Click on create a **New application**.
```
Comment (mteodor) [inline docs/continuous-delivery/gitops/connect-and-manage/use-flux.md:145]: how is this related to previous step `flamingo generate-app....`

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!
